components/Hero.js

```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link';
+
+export default function Hero({
+  title, description, bgImg, ctaText, ctaLink, socialMedia,
+}) {
+  return (
+    <section className="hero">
+      <img src={bgImg} alt="" className="hero-background" />
+
+      <div className="hero-shape" />
+
+      <div className="hero-content">
+        <h1 className="hero-title">{title}</h1>
+        <p className="hero-description">{description}</p>
+        <Link href={`/${ctaLink}`}>
+          <p className="button-main hero-button">{ctaText}</p>
+        </Link>
+
+        <div className="hero-media-wrapper">
+          <div className="hero-line" />
+          {socialMedia.map((item) => (
```

Comment:
we can use beacause we are not using item itself directly only his fields props
```suggestion
          {socialMedia.map(      ({ fields: { title, icon, link } })     ) => ( 
```
and at the bottom, you can access (title, icon, and link) directly.